### DISCUSSION TITLE
Installation MAC OS problem

### DISCUSSION THREAD

**Discussion by charlsdust** [publicsamples/Quetzalcoatl#14] [Q&A] [answered]
![image](https://github.com/user-attachments/assets/f0f04e99-b33a-47b4-9703-fc2fcef7a20a)
Hello,

After the plugin installation I couldn't find following file.

**Comment by RickTaylor79** [comment, marked as answer]
If you're seeing this, chances are you've not installed it correctly (I know, installation is a mess right now!). 

Make sure you've extracted the content files You'll need something that can extract multipart zips as MacOS doesn't recognise them by default, [the unarchiver](https://theunarchiver.com) works well for this.

For everything to work, your Quetzalcoatl folder should look like this:

<img width="728" alt="Screenshot 2024-07-20 at 14 02 19" src="https://github.com/user-attachments/assets/7bb084e8-719f-4b77-b474-4d357fc90420">

**Comment by charlsdust** [comment]
Thanks for the help. I've extracted the files and copied to the folder located in applicationsupport and now I can test the synth. Thanks for the help/

**Comment by charlsdust** [comment]
Thanks for the help. I've extracted the files and copied to the folder located in applicationsupport and now I can test the synth. Thanks for the help.

**Comment by GlobalSequence** [comment]
I'm getting an error that says "The File /Library/Application Support/modularsamples/Quetzalcoatl/AudioResources.dat can't be found. 

I had to download that file separately on the github and manually place it in the folder, but it's still not being recognized. I also don't have the LinkOSX file that appears in the screenshot above.

I'm testing in Ableton, so I hid the plugin window and showed it again and I can see the app. In the "Presets" menu, there is nothing showing under "Sample location:", so I tried setting the "Change sample folder location" to the "Samples" folder manually, but clicking "Open" simply makes the folder select window disappear and there is still no folder showing below "Sample location:"

This is on MacOS Monterey on an Intel Mac if that helps!

<img width="773" alt="Screen Shot 2024-07-23 at 12 51 24 AM" src="https://github.com/user-attachments/assets/677c7af7-2b97-40e5-809c-7070a822736e">

**Comment by RickTaylor79** [comment]
The content needs to be in the root library, not the user one. The full path should be:

Macintosh HD/Library/Application Support/modularsamples/Quetzalcoatl/

Hope this helps. In the next day or two I'll be releasing an update that makes installation much easier/

**Reply by GlobalSequence** [reply]
Solved!

**Comment by lonegud** [comment]
Hi,
when I copy and paste all the expansions(Hybrids,Misc Analogue, System100, Xpander, Roland Misc) into the Expansions folder, they don't appear when i open up the plugin.
I'm In ableton on mac

**Reply by RickTaylor79** [reply]
Can you send a screenshot of the expansions folder? For reference it should look like this:

<img width="937" alt="Screenshot 2024-08-01 at 12 27 33" src="https://github.com/user-attachments/assets/cc0bf779-57b4-4032-8cec-3d24bdb2877d">

Keep in mind the plugin needs to be restarted after the expansion are installed.

**Comment by gabeb8403** [comment]
I've gone through all the steps in the youtube installation video and this github written instruction. I keep getting stuck on the step before I drag files into the extension folder. I cannot locate any modularsamples folder in my application support folder despite installing the original sample file in the plug in in my daw.

**Reply by RickTaylor79** [reply]
Are you sure you're not looking in the user library? the full path is Macintosh HD/Library/Application Support/modularsamples/Quetzalcoatl.

If you're struggling to find it, try this. Open Quetzalcoatl and click on "Presets".  Then click the "More" menu and select "Show preset folder". The preset folder should now popup in the finder, this is a sub folder of Quetzalcoatl.

<img width="837" alt="Screenshot 2024-08-01 at 20 46 51" src="https://github.com/user-attachments/assets/f5af6f2f-a336-45f6-9443-362d16eff4f7">

**Reply by gabeb8403** [reply]
I followed this path but there is still no modularsamples folder. I can't open presets in quetzalcoatl either because the screen still looks like the photo below despite installing the samples per the written instructions and the youtube instructions.
![Screenshot 2024-08-01 at 8 14 39 PM](https://github.com/user-attachments/assets/9f836308-7c57-44ad-8639-ec7e90991cf2)

**Reply by RickTaylor79** [reply]
When you installed the .hr1 file, where did you install the samples to? try navigating to there using "choose sample folder".

**Reply by RickTaylor79** [reply]
Okay, can confirm there was a fatal flaw with the Mac side of things. See this post for a fix:

**https://github.com/publicsamples/Quetzalcoatl/discussions/18#discussioncomment-10222023**
